Sources_v3/StreamChatUI/Utils/UIConfigProvider.swift

```diff
@@ -72,23 +81,23 @@ extension UIConfigProvider where Self: UIView {
 
 public extension GenericUIConfigProvider where Self: UIViewController {
     func register<T: ExtraDataTypes>(config: UIConfig<T>) {
-        view.anyUIConfig = config
+        anyUIConfig = config
```

Comment:
How view will get vc uiconfig now?

Reply:
Actually if we go with `UIResponder` approach, then we can iterate over Responder chain via `UIResponder.next`

For `UIView` it will return either associated `UIViewController` if exist or `superview`
For `UIViewController`  it will return his view superview
https://developer.apple.com/documentation/uikit/uiresponder/1621099-next

Reply:
You're right, and the solution you've suggested would also solve passing vc's config to it's view.. I'll try it 👍 
cc @VojtaStavik